core/sys/linux/sys.odin

```diff
@@ -769,8 +783,7 @@ execve :: proc "contextless" (name: cstring, argv: [^]cstring, envp: [^]cstring)
 		ret := syscall(SYS_execve, cast(rawptr) name, cast(rawptr) argv, cast(rawptr) envp)
 		return Errno(-ret)
 	} else {
-		ret := syscall(SYS_execveat, AT_FDCWD, cast(rawptr) name, cast(rawptr) argv, cast(rawptr) envp)
-		return Errno(-ret)
+		return execveat(AT_FDCWD, name, argv, envp, 0)
```

Comment:
It's a small thing so I was wondering as to whether I want to make this remark, but I don't want to call procedures that call syscalls as opposed to calling syscalls directly, because they might have additional logic that maybe you don't want to duplicate. Let's keep this package as plain as possible